js/adapt-stateful-session.js

```diff
@@ -79,6 +79,24 @@ define([
 			this.listenTo(Adapt.blocks, 'change:_isComplete', this.onBlockComplete);
 			this.listenTo(Adapt.course, 'change:_isComplete', this.onCompletion);
 			this.listenTo(Adapt, 'assessment:complete', this.onAssessmentComplete);
+			this.listenTo(Adapt, 'app:resetSession', this.onResetSession);
```

Comment:
Fair enough.

I'm thinking this is more of a "notify:*" situation https://github.com/adaptlearning/adapt_framework/blob/master/src/core/js/notify.js#L54-L68 where the behaviour is triggered by the endpoint event listener or spoor provides an API for other plugins to use.

I can see that the ``languagePicker:changeTrigger`` is much the same as "accessibility:toggle" https://github.com/adaptlearning/adapt_framework/blob/master/src/core/js/views/accessibilityView.js#L46

Maybe we should have left it as "app:sessionReset" which is a pretty reasonable space between the two where neither languagePicker, nor spoor take ownership?